tests/acceptance/config/yaml-test.js

```diff
@@ -5,11 +5,17 @@ import { percySnapshot } from 'ember-percy';
 import signInUser from 'travis/tests/helpers/sign-in-user';
 import page from 'travis/tests/pages/build';
 
-let yaml = `
+let config = `
 language: jortle
 sudo: tortle
 `;
 
+let rawConfigs = [
+  {
+    config: config
+  }
+];
```

Comment:
It would also be nice to have a test for multiple configs - to verify they all rendered properly

Reply:
Thanks for reviewm @artursmirnov . Updates coming soon:)